packages/modules/desktop-viewer-react/src/types.ts

```diff
@@ -3,9 +3,9 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-import type { ViewerProps } from "@itwin/viewer-react";
+import type { FileViewerProps } from "@itwin/viewer-react";
 
-export type DesktopViewerProps = Omit<ViewerProps, "appInsightsKey">;
+export type DesktopViewerProps = Omit<FileViewerProps, "appInsightsKey">;
```

Comment:
the issue here is that currently the desktop viewer supports online viewing and we have product teams who are using that functionality. We have to think long and hard about taking this hard stance of only file support in the desktop. I understand that there are some that don't agree within online viewing from the desktop but is it in the best interest of our users to remove support completely?

Reply:
I thought with the latest desktop viewer we were always opening models againt the local backend, and never the hosted one. I do think we should support blank connections here. I wouldnt be opposed to connected models here either, but yea agreed, lets go over this in more detail with stakeholders

Reply:
Updated so desktop can open any kind of model